docs/using-lagoon-advanced/insights.md

```diff
@@ -0,0 +1,83 @@
+# Lagoon Insights
+
+*Warning, the Insights System is currently being reworked, so while most of this will still hold in some form, there may be some flux in terms of the details described below*
+
+Lagoon insights is a rough collection of data related to the state of an environment.
+
+
+# Facts
+
+The Facts API is used to store arbitrary data against an environment. It is something like a more structured key/value store.
+
+A fact consists of the following fields
+
+* id - generated when a fact is inserted.
+* environment - *NB* reference to the Environment targeted by the fact.
```

Comment:
I would maybe take out the NB just because not everyone knows what that stands for.